src/js_realm.hpp

```diff
@@ -977,5 +985,39 @@ void RealmClass<T>::compact(ContextType ctx, ObjectType this_object, Arguments a
     return_value.set(realm->compact());
 }
 
+#if REALM_ENABLE_SYNC
+template<typename T>
+void RealmClass<T>::subscribe_to_objects(ContextType ctx, ObjectType this_object, Arguments args, ReturnValue &return_value) {
+    args.validate_count(3);
+
+    SharedRealm realm = *get_internal<T, RealmClass<T>>(this_object);
+    auto class_name = Value::validated_to_string(ctx, args[0]);
+    auto query = Value::validated_to_string(ctx, args[1]);
+    auto callback = Value::validated_to_function(ctx, args[2]);
+
+    auto cb = [=](realm::Results results, std::exception_ptr err) {
+        if (err) {
+            try {
+                rethrow_exception(err);
+            }
+            catch (const std::exception& e) {
+                typename T::Value arguments[2];
+                arguments[0] = Value::from_null(ctx);
+                arguments[1] = Value::from_string(ctx, e.what());
+                Function<T>::callback(ctx, callback, this_object, 2, arguments);
```

Comment:
i think you need to capture the `ctx` as Protected<ContectType>, also the `this_object`

Reply:
unless this is called immediately before unwinding the stack